doc/errors.org

```diff
@@ -0,0 +1,95 @@
+* Errors
+https://ziglang.org/documentation/0.10.1/#Errors
```

Comment:
The last version of Zig is 0.11, is it voluntary to have the 0.10 one?

Reply:
same reponse as https://github.com/Pismice/HEIG_ZIG/pull/2#discussion_r1518894572

Reply:
see: [comment change link](https://github.com/Pismice/HEIG_ZIG/pull/2#discussion_r1518905377)